### PR TITLE
Turtle writer options

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: source/documentation/io/rdf-output.md
Comment (kinow) [inline source/documentation/io/rdf-output.md:247]: s/the settign/the setting

File: source/documentation/io/rdf-output.md
Comment (kinow) [inline source/documentation/io/rdf-output.md:254]: Ha, nice trick!

File: source/documentation/io/rdf-output.md
Comment (kinow) [inline source/documentation/io/rdf-output.md:281]: s/it's/its

**Review by kinow — APPROVED** [review]
Couple comments, no blockers :+1:

File: source/documentation/io/rdf-output.md
Comment (afs) [reply source/documentation/io/rdf-output.md:254]: Somewhat crude really! But to tweak the CSS, and then have it ripple out across the site is a somewhat bigger task.

**Review by rvesse — APPROVED** [review]

**Comment by kinow** [comment]
Are we waiting for a PR in Jena before merging this one @afs ?

**Comment by afs** [comment]
This is for 3.17.0 - I'll note when things need to wait for a release next time.

https://issues.apache.org/jira/browse/JENA-1968

**Comment by afs** [comment]
Applied locally from diff, edited to resolve the merge conflict.

**Pull request closed without merging**
